app/forms/developers_search_form.rb

```diff
@@ -0,0 +1,22 @@
+class DevelopersSearchForm
+  include Virtus.model
+  include ActiveModel::Model
+  include ActiveRecord::Sanitization::ClassMethods
+
+  attribute :email, String
```

Comment:
@truongle7793 

<img width="723" alt="screen shot 2018-04-04 at 14 28 54" src="https://user-images.githubusercontent.com/4110469/38294279-8bb6df3e-3814-11e8-9907-1905e4aa4c70.png">

Reply:
Thanks @thienhv 
I have remove email from the search form since it is not required.